conductr_cli/conduct_logs.py

```diff
@@ -13,27 +15,93 @@ def logs(args):
 
     log = logging.getLogger(__name__)
 
+    if args.follow:
+        old_data = []
+
+        while not follow_is_done():
```

Comment:
`follow_is_done()` currently always returns `False`:

```
ef follow_is_done():
    return False
```

This can't be right?!